src/smt/set_defaults.cpp

```diff
@@ -870,6 +870,27 @@ bool SetDefaults::usesSygus(const Options& opts) const
   return false;
 }
 
+bool SetDefaults::usesInputConversion(const Options& opts,
+                                      std::ostream& reason) const
+{
+  if (opts.smt.solveBVAsInt != options::SolveBVAsIntMode::OFF)
+  {
+    reason << "solveBVAsInt";
+    return true;
+  }
+  if (opts.smt.solveIntAsBV > 0)
```

Comment:
Why is this option a `uint64_t` rather than `bool` in the toml option configuration?

Reply:
The value is the bitwidth to try.